up.ts

```diff
@@ -114,16 +116,26 @@ const start = async function () {
     // wait till the deploy finishes
     await deployRegistry.status();
 
+    // Add the registry address to the Validator config
+    const configFilePath = join(VALIDATOR_DIR, 'local/api/config.json');
+    const validatorConfig = JSON.parse(
+        await Deno.readTextFile(configFilePath)
+    );
+    validatorConfig.Chains[0].Registry.ContractAddress = '0xe7f1725e7734ce288f8367e1bb143e90bb3f0512';
+
+    await Deno.writeTextFile(configFilePath, JSON.stringify(validatorConfig, null, 2));
```

Comment:
Figured we could save some future headaches if we automate setting the registry address in the validator config

Reply:
nice